plugins/spi/SPIDevice.h

```diff
@@ -86,7 +86,7 @@ class SPIDevice: public ola::Device {
   static const char SPI_DEVICE_NAME[];
   static const char HARDWARE_BACKEND[];
   static const char SOFTWARE_BACKEND[];
-  static const uint8_t MAX_GPIO_PIN = 25;
+  static const uint16_t MAX_GPIO_PIN = 25;
```

Comment:
So is the point of this to set which pins can be used as additional chip selects? If the goal is to let any GPIO pin be defined for SPI use, we should set it to (i believe) 1024 (as in kernel 4.4, CHIP GPIO starts at 1016).